### PR TITLE
Data Science with Python : Elbow Method In KMeans Clustering #4171

### LINKED ISSUE #4171: Data Science with Python : Elbow Method In KMeans Clustering
Welcome to 'DSWP' Team, good to see you here

With this issue, readers will get introduced with core information to Elbow Method In KMeans Clustering along with sample code completely in a effective way.

To get assigned to this issue, add your serial numbers mentioned in the spreadsheet of "Data Science with Python", the approach one would follow and choice you prefer (Documentation, Audio, Video). You can go with all three or any number of options you're interested to work on.

If you had referred any resources, add them up in "DS Resources". Similarly if you had used datasets, include them in "DS Datasets".

Domain : Machine Learning

Mentors Assigned : Dhruv Bajaj / Yash Goel

Points to Note :

The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.

"Issue Title" and "PR Title should be the same. Include issue number along with it.

Changes should be made inside the Datascience_With_Python/ directory & Datascience_With_Python branch.

Follow Contributing Guidelines & Code of Conduct before start Contributing.

This issue is only for 'GWOC' contributors of 'DSWP' domain.

All the best. Enjoy your open source journey ahead. 😎

### PR DESCRIPTION
**Pull request by ShubhamPatel01**
<hr>

## Description 📜

I prepared a video in which I covered topics which includes Introduction to Elbow method, Working of Elbow method and implementing it. Also, I have covered advantages and disadvantages of it.
Fixes #4171  

<hr>

## Type of change 📝

- [x] Video (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)

<hr>

## Domain of Contribution 📊

- [x] Datascience with Python

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Comment by ShubhamPatel01** [comment]
Please evaluate @prathimacode-hub
Name: Shubham Patel
Issue: #4171 
Type: Video
Batch number: 3
Serial number: 104

**Review by prathimacode-hub — CHANGES_REQUESTED** [review]
Don't add vimeo link directly. Instead add it up in Google Drive and share the link here in README.md @ShubhamPatel01

**Comment by ShubhamPatel01** [comment]
@prathimacode-hub mam, I have corrected the link.

File: Datascience_With_Python/Machine Learning/Videos/Elbow Method In KMeans Clustering/README.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+plt.show()
+```
+![Scatter Plot 1](./Images/Scatter_1.png)
+
+Selecting latitude and longitude columns to apply k means 
+```
+x=data.iloc[:,1:3]
+x.head()
+```
+![Data 1](./Images/Data_Head_1.png)
```
Comment (DhruvBajaj01) [inline Datascience_With_Python/Machine Learning/Videos/Elbow Method In KMeans Clustering/README.md:62]: please don't add codes in your readme.

Comment (ShubhamPatel01) [reply Datascience_With_Python/Machine Learning/Videos/Elbow Method In KMeans Clustering/README.md:62]: @DhruvBajaj01 , I am just following contribution guidelines which are pinned on discord channel.
Screenshot (1404)
Plus, I do think that these code sections are better for understanding and will help someone if he/she wants to try to implement anything as from the video, it is not easy to copy and implement it.
Thankyou

**Comment by KhushiKV** [comment]
Hi @prathimacode-hub , I would like to work on this issue.
Name: Khushi Verma
Batch: 16
S. No.:  568
Type: Documentation
Approach: I would start with ann introduction to K-meansClustering Methos, and then move on to Elbow Method. Beginning with what it is and how it is useful, I would like to emphasize on how it is linked to KMeans, along with its advantages to conclude.
Thank you.

**Comment by ShubhamPatel01** [comment]
> Hi @prathimacode-hub , I would like to work on this issue.
> Name: Khushi Verma
> Batch: 16
> S. No.: 568
> Type: Documentation
> Approach: I would start with ann introduction to K-meansClustering Methos, and then move on to Elbow Method. Beginning with what it is and how it is useful, I would like to emphasize on how it is linked to KMeans, along with its advantages to conclude.
> Thank you.

@KhushiKV , this is a PR, not an issue.

**Review by prathimacode-hub — APPROVED** [review]
Your PR approved. Impressive work. 🙌👌@ShubhamPatel01

**Pull request merged**